### ISSUE TITLE
Cannot reset password.

### ISSUE THREAD

**Issue by javadevmtl** [ASKBOT/askbot-devel#666]
Hello running 0.10.0 the site is working pretty well. Questions/Answers/Notifications user registration. All going well.

But my users are reporting that they cannot reset their password. Using standard email/password auth.

You can enter your email, which send out the recovery link, but once you reach the recovery link the below error is printed...

```
FieldError at /account/recover/
Cannot resolve keyword 'email_key' into field. Choices are: _message_set, activity, activityauditstatus, anonymousanswer, anonymousquestion, askbot_profile, award_user, badges, bulktagsubscription, created_tags, date_joined, deleted_posts, deleted_tags, draft_answers, draftquestion, email, endorsed_posts, first_name, followed_threads, followrecord, group_membership, group_messaging_sent_messages, groups, id, incoming_activity, is_active, is_staff, is_superuser, last_edited_posts, last_login, last_name, lastvisittime, locked_posts, logentry, messagememo, notification_subscriptions, password, postflagreason, postrevision, postrevisions, posts, question_views, replyaddress, repute, senderlist, suggested_tags, tag_selections, tag_synonyms, thread, unreadinboxcounter, unused_favorite_threads, unused_last_active_in_threads, user_favorite_questions, user_permissions, userassociation, username, userpasswordqueue, votes
Request Method:	GET
Request URL:	http://xxxxxx:8000/account/recover/?validation_code=3029a4cb75e23c2fe470ye492bf6d413
Django Version:	1.8.16
Exception Type:	FieldError
```

**Comment by evgenyfadeev** [comment]
It's now fixed in the master branch and release 0.10.1 thanks!

**Comment by javadevmtl** [comment]
Is it a big change? I would like to try to apply a patch if it's something small...
